spec.html

```diff
@@ -1094,7 +1095,7 @@ <h1>
           1. Let _len_ be the length of _string_.
           1. If _searchValue_ is the empty String and _fromIndex_ &le; _len_, return _fromIndex_.
           1. Let _searchLen_ be the length of _searchValue_.
-          1. For each integer _i_ starting with _fromIndex_ such that _i_ &le; _len_ - _searchLen_, in ascending order, do
+          1. For each integer _i_ in the inclusive interval from _fromIndex_ to _len_ - _searchLen_, in ascending order, do
```

Comment:
This doesn't seem like an improvement but I guess I'm fine with it.

Reply:
I'd prefer to be consistent in using intervals with these "for each" loops unless there's good reason not to.

Reply:
Changed all the loops over intervals to use chained comparison instead.